src/editorial/web/components/starRating/starRating.stories.tsx

```diff
@@ -0,0 +1,5 @@
+export * from './stories/default';
+
+export default {
+	title: 'Star Rating',
+};
```

Comment:
I guess if there's only one set of stories here there's no need too abstract them into another file?

Reply:
Unsure. I was attempting to follow the convention used for the [`core` components](https://github.com/guardian/source/tree/main/src/core/components). If it's not quite right happy to change.